api/repo/repair.go

```diff
@@ -62,6 +62,8 @@ import (
 
 // RepairRepo represents the API handler to remove
 // and then create a webhook for a repo.
+//
+//nolint:funlen // ignore statement count
 func RepairRepo(c *gin.Context) {
```

Comment:
Do we want to add similar logic to the `api/scm/sync` API handlers?

Reply:
yeah good call